### ISSUE TITLE
What is the meaning behind the points the PID returns?

### ISSUE THREAD

**Issue by dmshimself** [soloam/ha-pid-controller#36]
I can see that the PID produces values on my system of numbers between 0.1, 1.0, 10, 15 and 35.  What do these different numbers mean and what is the maximum and minimum values?

I'm guessing the bigger the number, the more power you would like the heating or cooling element to push into the system.  I usually have a simple heater which is either on or off, but I now have access to a controller which can deliver anything from 0 to 100% of the power, so I wanted to find out if it's possible to use sensibly that along with the value of the PID points.

**Comment by dmshimself** [comment]
Does anyone know?

**Comment by soloam** [comment]
The output of the PID is between the minimum and maximum that you set, so if your system is on and off you should have a minimum of 0 and a maximum of 1, and no decimals. So you would only have a output off 0 (off) and 1 (on).

If for example your heater would have five modes of speed, you would set a minimum of 0 and a maximum of 5, and a decimal of 0. This would be 0 (off) and 1-5 (the modes to set the heater).

If for example your heater is able to output a variable temperature from 0º to 35º and with a precision of 1 decimal. You would set the minimum to 0, the maximum to 35 and the decimal to 1. The output would be the temperature to set the heater.

Basically the output is action that the PID does to change the input.

**Comment by dmshimself** [comment]
Perfect - many thanks
